src/main/java/lv/ctco/springboottemplate/features/todo_statistics/dto/TodoStatisticsRequest.java

```diff
@@ -0,0 +1,42 @@
+package lv.ctco.springboottemplate.features.todo_statistics.dto;
+
+import io.swagger.v3.oas.annotations.Parameter;
+import java.time.Instant;
+import lv.ctco.springboottemplate.features.todo_statistics.TodoStatisticsFormatEnum;
+import lv.ctco.springboottemplate.features.todo_statistics.validators.TodoStatisticsValidDateRange;
+
+@TodoStatisticsValidDateRange
+public class TodoStatisticsRequest {
+  @Parameter(example = "2025-09-01T10:30:00Z")
+  private Instant from;
```

Comment:
Consider switching to LocalDateTime, in case you don't care about server time or timezone issue in this example